src/platforms/platform.js

```diff
@@ -1,22 +1,13 @@
 'use strict';
 
-import helpers from '../helpers/index';
-import basic from './platform.basic';
-import dom from './platform.dom';
-
-// @TODO Make possible to select another platform at build time.
-const implementation = dom._enabled ? dom : basic;
-
 /**
- * @namespace Chart.platform
- * @see https://chartjs.gitbooks.io/proposals/content/Platform.html
- * @since 2.4.0
+ * Abstract class that allows abstracting platform dependencies away from the chart.
  */
-export default helpers.extend({
+export default class Platform {
 	/**
-	 * @since 2.7.0
+	 * @param {object} options - The chart options
```

Comment:
Remove `@param` tag since there's no parameters

Reply:
I'll change this to be clear that this param is passed into child constructors